### ISSUE TITLE
Problem with cell wrapping 

### ISSUE THREAD

**Issue by mchusovlianov** [unidoc/unidoc#169]
For some reason, this table has the empty second field. Could you check whats wrong here, please?
As regular font we use Lato Regular TTF font.
```
pageHistoryTable := creator.NewTable(4)
	pageHistoryTable.SetColumnWidths(0.1, 0.6, 0.15, 0.15)
	content := [][]string{
		{"1", "FullText Search Highlight the Term in Results ·\n\nissues 60", "120", "130"},
		{"1", "FullText Search Highlight the Term in Results ·\n\nissues 60", "120", "130"},
		{"1", "FullText Search Highlight the Term in Results ·\n\nissues 60", "120", "130"},
		{"1", "FullText Search Highlight the Term in Results ·\n\nissues 60", "120", "130"},
		{"1", "FullText Search Highlight the Term in Results ·\n\nissues 60", "120", "130"},
	}
	for _, rows := range content {
		for _, txt := range rows {
			p := creator.NewParagraph(txt)
			p.SetFontSize(12)
			p.SetFont(r.regularFont)
			p.SetColor(creator.ColorBlack)
			p.SetEnableWrap(true)


			cell := pageHistoryTable.NewCell()
			cell.SetBorder(creator.CellBorderStyleBox, 1)
			cell.SetContent(p)
		}
	}

	r.c.Draw(pageHistoryTable)
```

**Comment by mchusovlianov** [comment]
@gunnsth Thanks a lot! I'll check it. Is it possible to remove only one symbol but display string?

**Comment by gunnsth** [comment]
Closed by merge of PR #179 into master (and release of 2.1.1).
